addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -184,40 +232,67 @@ function factory(dependencies) {
             if (!isOdooFocused) {
                 this._notifyNewChannelMessageWhileOutOfFocus({ channel, message });
             }
+            if (channel.model === 'mail.channel') {
+                await channel.markAsFetched();
```

Comment:
Remove await from here, rest of the code can execute even if rpc is pending/fails.